apps/webservice/src/app/[workspaceSlug]/_components/relationships/RelationshipsDiagramDependencies.tsx

```diff
@@ -0,0 +1,419 @@
+        )}
+        style={selectedStyle}
+      >
+        {isSelected && <div className="animated-border" />}
+        <div className="flex items-center gap-2">
+          <TargetIcon version={data.version} kind={data.kind} />
+          <span className="text-xs">{data.kind}</span>
+        </div>
+        <div className="text-sm">{data.name}</div>
+      </div>
+
+      <Handle
+        type="target"
+        className={cn(
+          "h-2 w-2 rounded-full border border-neutral-500 bg-neutral-800",
+          isKubernetes && "border-blue-500/70",
+          isTerraform && "border-purple-500/70",
+          isSharedCluster && "border-blue-500/70",
+        )}
+        position={Position.Bottom}
+      />
+      <Handle
+        type="source"
+        className={cn(
+          "h-2 w-2 rounded-full border border-neutral-500 bg-neutral-800",
+          isKubernetes && "border-blue-500/70",
+          isTerraform && "border-purple-500/70",
+          isSharedCluster && "border-blue-500/70",
+        )}
+        position={Position.Top}
+      />
+      <style jsx>{`
+        .animated-border {
+          position: absolute;
+          top: -1px;
+          left: -1px;
+          right: -1px;
+          bottom: -1px;
+          background-image: linear-gradient(
+              90deg,
+              ${animatedBorderColor} 50%,
+              transparent 50%
+            ),
+            linear-gradient(90deg, ${animatedBorderColor} 50%, transparent 50%),
+            linear-gradient(0deg, ${animatedBorderColor} 50%, transparent 50%),
+            linear-gradient(0deg, ${animatedBorderColor} 50%, transparent 50%);
+          background-repeat: repeat-x, repeat-x, repeat-y, repeat-y;
+          background-size:
+            20px 1px,
+            20px 1px,
+            1px 20px,
+            1px 20px;
+          background-position:
+            0 0,
+            0 100%,
+            0 0,
+            100% 0;
+          border-radius: calc(0.5rem - 2px);
+          animation: moveDashedBorder 1s linear infinite;
+          pointer-events: none;
+        }
+
+        @keyframes moveDashedBorder {
+          0% {
+            background-position:
+              0 0,
+              0 100%,
+              0 0,
+              100% 0;
+          }
+          100% {
+            background-position:
+              20px 0,
+              -20px 100%,
+              0 -20px,
+              100% 20px;
+          }
+        }
+      `}</style>
+    </>
+  );
+};
+
+const DepEdge: React.FC<EdgeProps> = ({
+  sourceX,
+  sourceY,
+  targetX,
+  targetY,
+  sourcePosition,
+  targetPosition,
+  label,
+  style = {},
+  markerEnd,
+}) => {
+  const [edgePath, labelX, labelY] = getBezierPath({
+    sourceX,
+    sourceY,
+    sourcePosition,
+    targetX,
+    targetY,
+    targetPosition,
+  });
+
+  return (
+    <>
+      <BaseEdge
+        path={edgePath}
+        markerStart={markerEnd}
+        style={{ strokeWidth: 2, ...style }}
+      />
```

Comment:
these are intrinsict attributes of the react flow base edge so we cannot rename or change it, and we need the arrows flipped since we do "source depends on target" logic